packages/react-core/src/components/Button/__tests__/Button.test.tsx

```diff
@@ -55,10 +55,9 @@ test('Renders with class pf-m-block when isBlock = true', () => {
   expect(screen.getByRole('button')).toHaveClass('pf-m-block');
 });
 
-// TODO:  Reenable or remove with issue #9907
-xtest('Renders with class pf-m-active when isActive = true', () => {
-  render(<Button isActive>Active Button</Button>);
-  expect(screen.getByRole('button')).toHaveClass('pf-m-active');
+xtest('Renders with class pf-m-clicked when isClicked = true', () => {
```

Comment:
Can we reenable this one?

Reply:
Well, there will no longer be the `isActive` prop and the `pf-m-active` modifier in V6, it is renamed to `isClicked`. So I don't think we should reenable that test.

Reply:
It looks like you modified it for the `isClicked`.  Does the test fail?  I that why it is still disabled?

Reply:
I have opened this issue https://github.com/patternfly/patternfly-react/issues/9989

Reply:
Oh, I did not realize that when it is marked as `xtest` and not `test` that it means the test is disabled.

Reply:
fix PR: https://github.com/patternfly/patternfly-react/pull/10013